src/modules/musig/vectors.h

```diff
@@ -0,0 +1,346 @@
+/**
+ * Automatically generated by ./contrib/musig2-vectors.py.
```

Comment:
I think this was lost in some rebase. 

```suggestion
 * Automatically generated by tools/test_vectors_musig2_generate_.py.
```

And just remove `contrib/musig2-vectors.py`, the PR currently adds the (identical) script to both paths.

Reply:
Not a rebase issue, just confusion: I had thought that I had forgotten to add the python script because it's called differently in -zkp.

Reply:
fixed